app/src/main/kotlin/org/vinaygopinath/launchchat/screens/main/MainActivity.kt

```diff
@@ -99,6 +108,16 @@ class MainActivity : AppCompatActivity() {
         }
     }
 
+    private fun showPhoneNumberSelectionDialog(phoneNumbers: List<String>, onNumberSelected: (String) -> Unit) {
+        val items = phoneNumbers.toTypedArray()
+        AlertDialog.Builder(this)
```

Comment:
This dialog does not seem to be showing an explanatory _message_, which is one of the acceptance criteria defined in #24